guides/plugins/plugins/storefront/add-custom-page.md

```diff
@@ -162,6 +162,23 @@ class ExampleController extends StorefrontController
 
 Note, that we've added the page to the template variables.
 
+#### Adjusting the services.xml
+
+In addition, it is necessary to pass the argument with the ID of the `ExamplePageLoader` class to the [configuration ](add-custom-controller#services-xml-example) of the controller service in the `services.xml`.
+
+```html
```

Comment:
shouldn't that be 

```suggestion
```xml
```

Reply:
I was wondering why on some pages code blocks for xml files you are using `xml` and on some `html`.
As I compared the pages I was thinking it is consistent using `html` for xml files not including the full xml markup.

Can you confirm this? Else there are many pages with wrong code block tags using html for xml files.

Reply:
If you want to fix it for all occurrences, I would appreciate it, but if not, no worries, we can keep it like that for consistency

Reply:
Lets say we keep it for this PR and I will check on how many blocks the tag is wrong and see then later today?

Reply:
totally fine for me :ok_hand: